.travis.yml

```diff
@@ -9,6 +9,9 @@ sudo: false
 matrix:
   allow_failures:
     - rvm: ruby-head
+  exclude:
+    - rvm: 2.3.3
```

Comment:
I think the commit message should be 2.3?

Reply:
Oy Vey! Good catch :)